capable/Cargo.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "mc-sgx-capable"
+version = "0.1.0"
+edition = "2021"
+authors = ["MobileCoin"]
```

Do we want the other values populated here
```suggestion
authors = ["MobileCoin"]
rust-version = "1.62.1"
license = "Apache-2.0"
readme = "README.md"
repository = "https://github.com/mobilecoinfoundation/sgx"
```

---

Yeah, I basically ignored this because this crate isn't currently built out yet.